mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -1167,6 +1171,9 @@ pub struct TakerRequest {
     #[serde(default)]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub rel_protocol_info: Option<Vec<u8>>,
+    #[serde(default)]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub swap_version: Option<u32>,
```

Comment:
what about having this field a u32 defaulting to 1 if not present?
less Somes to unwrap

Reply:
Changed to u32 type https://github.com/KomodoPlatform/komodo-defi-framework/pull/2261/commits/707946a49fee99d7a074a0389bd4d4c10f155dbb

serialisation should also be skipped if 1u32 value was provided

Reply:
i don't think old nodes will fail to deserialize a message with an extra unknown field, but either way works.